core/src/main/java/tachyon/UnderFileSystem.java

```diff
@@ -62,6 +62,10 @@ public static UnderFileSystem get(String path) {
    * @return null for any unknown scheme.
    */
   public static UnderFileSystem get(String path, Object conf) {
+    if (path == null) {
+      return null;
```

Comment:
I would prefer to throw RuntimeException or IllegalArugmentException rather than quietly return null.